From 87bd93d63161836b0264afcda5a6d245f2ebe1ff Mon Sep 17 00:00:00 2001
From: Dave Airlie <airlied@redhat.com>
Date: Thu, 1 Sep 2016 06:16:12 +1000
Subject: [PATCH 476/532] Revert "drm: make DRI1 drivers depend on BROKEN"

commit 2b2fd56d7e92f134ecaae5c89e20f64dd0f95aa2 from
https://github.com/Xilinx/linux-xlnx.git

This reverts commit d10571fc4fba7d57fb8157f0be61dcbe4a7965ca.

This isn't how we get to do this unfortunately.

Signed-off-by: Dave Airlie <airlied@redhat.com>
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 drivers/gpu/drm/Kconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/gpu/drm/Kconfig b/drivers/gpu/drm/Kconfig
index 09c6f59c..af3db92 100644
--- a/drivers/gpu/drm/Kconfig
+++ b/drivers/gpu/drm/Kconfig
@@ -246,7 +246,6 @@ source "drivers/gpu/drm/mediatek/Kconfig"
 menuconfig DRM_LEGACY
 	bool "Enable legacy drivers (DANGEROUS)"
 	depends on DRM
-	depends on BROKEN
 	help
 	  Enable legacy DRI1 drivers. Those drivers expose unsafe and dangerous
 	  APIs to user-space, which can be used to circumvent access
-- 
2.7.4

